### PR TITLE
Packaging Flashtext extension

### PR DESCRIPTION
**Pull request by teolemon**
### What
- Brute-force extend the number of French recycling instructions we can extract based on CITEO guidelines
### Part of
- #600

### CODE REVIEW

File: data/ocr/packaging.txt [hunk 1]
```diff
@@ -3,24 +3,51 @@
...
+fr:Aérosol métal à recycler
+fr:Ailettes papier à recycler
+fr:Ailettes plastique à jeter
+fr:Ampoule verre à recycler
+fr:Applicateur carton à recycler
+fr:Applicateur plastique à jeter
+fr:Bac autres à jeter
+fr:Bac carton à recycler
+fr:Bac métal à recycler
+fr:Bag In Box (BIB) boîte carton à recycler, poche plastique à jeter
```
Comment (teolemon) [inline data/ocr/packaging.txt:23]:
```suggestion
fr:Bag In Box
fr:boîte carton à recycler
fr:poche plastique à jeter
```

File: data/ocr/packaging.txt [hunk 2]
```diff
@@ -3,24 +3,51 @@
...
+fr:Bandelette autres à jeter
```
Comment (teolemon) [inline data/ocr/packaging.txt:24]:
```suggestion
```

File: data/ocr/packaging.txt [hunk 3]
```diff
@@ -3,24 +3,51 @@
...
+fr:Bandelette carton à recycler
+fr:Bandelette métal à recycler
+fr:Bandelette papier à recycler
+fr:Barquette autres à jeter
```
Comment (teolemon) [inline data/ocr/packaging.txt:28]:
```suggestion
fr:Barquette à jeter
```

File: data/ocr/packaging.txt (hunk 2 above)
Comment (teolemon) [inline data/ocr/packaging.txt:24]:
```suggestion
fr:Bandelette à jeter
```

File: data/ocr/packaging.txt [hunk 4]
```diff
@@ -3,24 +3,51 @@
...
+fr:Barquette carton à recycler
 fr:Barquette en carton||barquette carton
+fr:Barquette en plastique||barquette plastique
+fr:Barquette métal à recycler
 fr:Barquette;fr:Film en plastique||barquette et film plastique
 fr:Barquette;fr:Opercule en plastique||barquette et opercule plastique
-fr:Barquette en plastique||barquette plastique
+fr:Bâtonnet autres à jeter
```
Comment (teolemon) [inline data/ocr/packaging.txt:35]:
```suggestion
fr:Bâtonnet à jeter
```

File: data/ocr/packaging.txt [hunk 5]
```diff
@@ -3,24 +3,51 @@
...
+fr:Bidon, Cubitainer plastique à recycler
+fr:Blister, coque avec chevalet carton
+fr:Blister, coque avec chevalet carton
+fr:Blister, coque carton à recycler
+fr:Blister, coque métal à recycler
+fr:Blister, coque plastique à jeter
+fr:Bobine, rouleau, mandrin autres à jeter
+fr:Bobine, rouleau, mandrin carton à recycler
+fr:Bobine, rouleau, mandrin métal à recycler
+fr:Bobine, rouleau, mandrin papier à recycler
```
Comment (teolemon) [inline data/ocr/packaging.txt:48]:
```suggestion
fr:Bidon métal à recycler
fr:Cubitainer métal à recycler
fr:Bidon plastique à recycler
fr:Cubitainer plastique à recycler
fr:Blister avec chevalet carton
fr:coque avec chevalet carton
fr:Blister avec chevalet carton
fr:coque avec chevalet carton
fr:Blister carton à recycler
fr:coque carton à recycler
fr:Blister métal à recycler
fr:coque métal à recycler
fr:Blister plastique à jeter
fr:coque plastique à jeter
fr:Bobine à jeter
fr:rouleau à jeter
fr:mandrin à jeter
fr:Bobine carton à recycler
fr:rouleau carton à recycler
fr:mandrin carton à recycler
fr:Bobine métal à recycler
fr:rouleau métal à recycler
fr:mandrin métal à recycler
fr:Bobine papier à recycler
fr:rouleau papier à recycler
fr:mandrin papier à recycler
```

File: data/ocr/packaging.txt [hunk 6]
```diff
@@ -29,40 +56,188 @@ fr:Boîte en métal||boîte metal
...
 fr:Boîte en plastique||boîte plastique
+fr:Boîte, boîtier autres à jeter
+fr:Boîte, boîtier carton à recycler
+fr:Boîte, boîtier métal à recycler
+fr:Boîte, boîtier plastique à jeter
+fr:Boîte, boîtier verre à recycler
+fr:Bol, coupe, coupelle carton à recycler
+fr:Bol, coupe, coupelle métal à recycler
+fr:Bol, coupe, coupelle plastique à jeter
+fr:Bol, coupe, coupelle verre à recycler
```
Comment (teolemon) [inline data/ocr/packaging.txt:67]:
```suggestion
fr:Boîte à jeter
fr:boîtier à jeter
fr:Boîte carton à recycler
fr:boîtier carton à recycler
fr:Boîte métal à recycler
fr:boîtier métal à recycler
fr:Boîte plastique à jeter
fr:boîtier plastique à jeter
fr:Boîte verre à recycler
fr:boîtier verre à recycler
fr:Bol carton à recycler
fr:coupe carton à recycler
fr:coupelle carton à recycler
fr:Bol métal à recycler
fr:coupe métal à recycler
fr:coupelle métal à recycler
fr:Bol plastique à jeter
fr:coupe plastique à jeter
coupelle plastique à jeter
fr:Bol verre à recycler
fr:coupe verre à recycler
fr:coupelle verre à recycler
```

File: data/ocr/packaging.txt [hunk 7]
```diff
@@ -29,40 +56,188 @@ fr:Boîte en métal||boîte metal
...
+fr:Cageot, cagette carton à recycler
+fr:Caisse, caissette
+fr:Caisse, caissette autres à jeter
+fr:Caisse, caissette carton à recycler
+fr:Caisse, caissette métal à recycler
+fr:Caisse, caissette papier à recycler
+fr:Cale, coin, cornière, éléments de bourrage, de mise en forme ou de maintien des produits autres à jeter
+fr:Cale, coin, cornière, éléments de bourrage, de mise en forme ou de maintien des produits carton à recycler
+fr:Cale, coin, cornière, éléments de bourrage, de mise en forme ou de maintien des produits métal à recycler
+fr:Cale, coin, cornière, éléments de bourrage, de mise en forme ou de maintien des produits plastique à jeter
```
Comment (teolemon) [inline data/ocr/packaging.txt:96]:
```suggestion
fr:Cale à jeter
fr:coin à jeter
fr:cornière à jeter
fr:éléments de bourrage à jeter
fr:éléments de mise en forme à jeter
fr:éléments de maintien des produits à jeter
fr:Cale à recycler
fr:coin à recycler
fr:cornière à recycler
fr:éléments de bourrage à recycler
fr:éléments de mise en forme à recycler
fr:éléments de maintien des produits à recycler
fr:Cale métal à recycler
fr:coin métal à recycler
fr:cornière métal à recycler
fr:éléments de bourrage métal à recycler
fr:éléments de mise en forme métal à recycler
fr:éléments de maintien des produits métal à recycler
fr:Cale carton à recycler
fr:coin carton à recycler
fr:cornière carton à recycler
fr:éléments de bourrage carton à recycler
fr:éléments de mise en forme carton à recycler
fr:éléments de maintien des produits carton à recycler
fr:Cale plastique à jeter
fr:coin plastique à jeter
fr:cornière plastique à jeter
fr:éléments de bourrage plastique à jeter
fr:éléments de mise en forme plastique à jeter
fr:éléments de maintien des produits plastique à jeter
```

File: data/ocr/packaging.txt [hunk 8]
```diff
@@ -29,40 +73,227 @@ fr:Boîte en métal||boîte metal
...
+fr:Bouteille||embouteillé
+fr:Brique (et son bouchon) carton à recycler
 fr:Brique en carton||brique carton
+fr:Cageot, cagette autres à jeter
+fr:Cageot, cagette carton à recycler
+fr:Caisse, caissette
+fr:Caisse, caissette autres à jeter
+fr:Caisse, caissette carton à recycler
+fr:Caisse, caissette métal à recycler
+fr:Caisse, caissette papier à recycler
```
Comment (teolemon) [inline data/ocr/packaging.txt:122]:
```suggestion
fr:Cageot à jeter
fr:cagette à jeter
fr:Cageot carton à recycler
fr:cagette carton à recycler
fr:Caisse
fr:caissette
fr:Caisse à jeter
fr:caissette à jeter
fr:Caisse carton à recycler
fr:caissette carton à recycler
fr:Caisse métal à recycler
fr:caissette métal à recycler
fr:Caisse papier à recycler
fr:caissette papier à recycler
```

File: data/ocr/packaging.txt [hunk 9]
```diff
@@ -29,40 +73,227 @@ fr:Boîte en métal||boîte metal
...
 fr:Capsule en métal||capsule métal
 fr:Capsule en métal||capsules metal
 fr:Capsule en métal||capsules métal
+fr:capsule métal
+fr:Capsule métal à recycler
+fr:Cartouche plastique à jeter
+fr:Cavalier, étui, fourreau,cornet autres à jeter
+fr:Cavalier, étui, fourreau,cornet carton à recycler
+fr:Cavalier, étui, fourreau,cornet métal à recycler
+fr:Cavalier, étui, fourreau,cornet papier à recycler
```
Comment (teolemon) [inline data/ocr/packaging.txt:165]:
```suggestion
fr:Cavalier à jeter
fr:étui à jeter
fr:fourreau à jeter
fr:cornet à jeter
fr:Cavalier carton à recycler
fr:étui carton à recycler
fr:fourreau carton à recycler
fr:cornet carton à recycler
fr:Cavalier métal à recycler
fr:étui métal à recycler
fr:fourreau métal à recycler
fr:cornet métal à recycler
fr:Cavalier papier à recycler
fr:étui papier à recycler
fr:fourreau papier à recycler
fr:cornet papier à recycler
```

File: data/ocr/packaging.txt [hunk 10]
```diff
@@ -29,40 +73,227 @@ fr:Boîte en métal||boîte metal
...
+fr:Enveloppe papier à recycler
+fr:Étiquette autres à jeter
+fr:Étiquette carton à recycler
+fr:Étiquette papier à recycler
 fr:Etui en carton||etui carton
 fr:Etui en carton||étui carton
+fr:Feuille d’emballage (papier kraft, papier sulfurisé, papier cadeau...), feuille plaquette (carton ondulé, fibre) autres à jeter
+fr:Feuille d’emballage (papier kraft, papier sulfurisé, papier cadeau...), feuille plaquette (carton ondulé, fibre) carton à recycler
+fr:Feuille d’emballage (papier kraft, papier sulfurisé, papier cadeau...), feuille plaquette (carton ondulé, fibre) métal à recycler
+fr:Feuille d’emballage (papier kraft, papier sulfurisé, papier cadeau...), feuille plaquette (carton ondulé, fibre) papier à recycler
```
Comment (teolemon) [inline data/ocr/packaging.txt:201]: TODO:check
```suggestion
fr:Feuille d’emballage
fr:papier kraft
fr:papier sulfurisé
fr:papier cadeau
fr:feuille plaquette
fr:carton ondulé
fr:fibre à jeter
fr:Feuille d’emballage carton à recycler
fr:papier kraft
fr:papier sulfurisé
fr:papier cadeau
fr:feuille plaquette carton à recycler
fr:carton ondulé
fr:fibre carton à recycler
fr:Feuille d’emballage métal à recycler
fr:papier kraft métal à recycler
fr:papier sulfurisé métal à recycler
fr:papier cadeau métal à recycler
fr:feuille plaquette métal à recycler
fr:carton ondulé métal à recycler
fr:fibre métal à recycler
fr:Feuille d’emballage papier à recycler
fr:papier kraft papier à recycler
fr:papier sulfurisé papier à recycler
fr:papier cadeau papier à recycler
fr:feuille plaquette papier à recycler
fr:carton ondulé papier à recycler
fr:fibre papier à recycler
```

File: data/ocr/packaging.txt [hunk 11]
```diff
@@ -29,40 +73,227 @@ fr:Boîte en métal||boîte metal
...
+fr:Terrine grès à jeter
+fr:Tonneau, tonnelet, fût autres à jeter
+fr:Tonneau, tonnelet, fût carton à recycler
+fr:Tonneau, tonnelet, fût métal à recycler
+fr:Tonneau, tonnelet, fût plastique à recycler
+fr:Tube rigide, tube souple, tube-bâton autres à jeter
+fr:Tube rigide, tube souple, tube-bâton carton à recycler
+fr:Tube rigide, tube souple, tube-bâton métal à recycler
+fr:Tube rigide, tube souple, tube-bâton plastique à jeter
+fr:Tube rigide, tube souple, tube-bâton verre à recycler
```
Comment (teolemon) [inline data/ocr/packaging.txt:293]:
```suggestion
fr:Tube rigide à jeter
fr:tube souple à jeter
fr:tube-bâton à jeter
fr:Tube rigide carton à recycler
fr:tube souple carton à recycler
fr:tube-bâton carton à recycler
fr:Tube rigide métal à recycler
fr:tube souple métal à recycler
fr:tube-bâton métal à recycler
fr:Tube rigide plastique à jeter
fr:tube souple plastique à jeter
fr:tube-bâton plastique à jeter
fr:Tube rigide verre à recycler
fr:tube souple verre à recycler
fr:tube-bâton verre à recycler
```

File: data/ocr/packaging.txt [hunk 12]
```diff
@@ -29,40 +73,227 @@ fr:Boîte en métal||boîte metal
...
+fr:Pot métal à recycler
+fr:Pot plastique à jeter
+fr:Pot verre à recycler
+fr:Présentoir, socle, casier, chevalet autres à jeter
+fr:Présentoir, socle, casier, chevalet carton à recycler
+fr:Présentoir, socle, casier, chevalet métal à recycler
+fr:Quadrette en plastique||quadrette plastique
+fr:Sac, sachet, poche, pochette autres à jeter
+fr:Sac, sachet, poche, pochette métal à recycler
+fr:Sac, sachet, poche, pochette papier à recycler
```
Comment (teolemon) [inline data/ocr/packaging.txt:276]:
```suggestion
fr:Sac à jeter
fr:sachet à jeter
fr:poche à jeter
fr:pochette à jeter
fr:Sac métal à recycler
fr:sachet métal à recycler
fr:poche métal à recycler
fr:pochette métal à recycler
fr:Sac papier à recycler
fr:sachet papier à recycler
fr:poche papier à recycler
fr:pochette papier à recycler
```

File: data/ocr/packaging.txt [hunk 13]
```diff
@@ -29,40 +73,227 @@ fr:Boîte en métal||boîte metal
...
 fr:Opercule en métal||opercule metal
 fr:Opercule en métal||opercule métal
 fr:Opercule en plastique||opercule plastique
-Pak-Lok||pak lok
-Pak-Lok||pak-lok
+fr:Opercule, fermeture barquette, fermeture blister autres à jeter
+fr:Opercule, fermeture barquette, fermeture blister carton à recycler
+fr:Opercule, fermeture barquette, fermeture blister métal à recycler
+fr:Opercule, fermeture barquette, fermeture blister papier à recycler
+fr:Opercule, fermeture barquette, fermeture blister plastique à jeter
```
Comment (teolemon) [inline data/ocr/packaging.txt:253]:
```suggestion
fr:Opercule à jeter
fr:fermeture barquette à jeter
fr:fermeture blister à jeter
fr:Opercule carton à recycler
fr:fermeture barquette carton à recycler
fr:fermeture blister carton à recycler
fr:Opercule métal à recycler
fr:fermeture barquette métal à recycler
fr:fermeture blister métal à recycler
fr:Opercule papier à recycler
fr:fermeture barquette papier à recycler
fr:fermeture blister papier à recycler
fr:Opercule plastique à jeter
fr:fermeture barquette plastique à jeter
fr:fermeture blister plastique à jeter
```

File: data/ocr/packaging.txt [hunk 14]
```diff
@@ -29,40 +73,275 @@ fr:Boîte en métal||boîte metal
...
+fr:Opercule plastique à jeter
+fr:fermeture barquette plastique à jeter
+fr:fermeture blister plastique à jeter
+fr:Panier osier à jeter
+fr:Panier plastique à jeter
 fr:Plaque en carton||plaque carton
+fr:Plat, plateau, plateau alvéolaire, plaque, plaquette autres à jeter
+fr:Plat, plateau, plateau alvéolaire, plaque, plaquette carton à recycler
+fr:Plat, plateau, plateau alvéolaire, plaque, plaquette métal à recycler
+fr:Plat, plateau, plateau alvéolaire, plaque, plaquette papier à recycler
```
Comment (teolemon) [inline data/ocr/packaging.txt:289]:
```suggestion
fr:Plat à jeter
fr:plateau à jeter
fr:plateau alvéolaire à jeter
fr:plaque à jeter
fr:plaquette à jeter
fr:Plat carton à recycler
fr:plateau carton à recycler
fr:plateau alvéolaire carton à recycler
fr:plaque carton à recycler
fr:plaquette carton à recycler
fr:Plat métal à recycler
fr:plateau métal à recycler
fr:plateau alvéolaire métal à recycler
fr:plaque métal à recycler
fr:plaquette métal à recycler
fr:Plat papier à recycler
fr:plateau papier à recycler
fr:plateau alvéolaire papier à recycler
fr:plaque papier à recycler
fr:plaquette papier à recycler
```

File: data/ocr/packaging.txt [hunk 15]
```diff
@@ -29,40 +73,275 @@ fr:Boîte en métal||boîte metal
...
 fr:Film en plastique;fr:Sachet en plastique||film et sachet plastique
 fr:Film en plastique||film plastique
+fr:Film, film à bulles plastique à jeter
+fr:Flacon (et son bouchon) métal à recycler
+fr:Flacon (et son bouchon) plastique à recycler
+fr:Flacon (et son bouchon) verre à recycler
+fr:Flacon applicateur, flacon pompe, flacon pistolet/spray métal à recycler
+fr:Flacon applicateur, flacon pompe, flacon pistolet/spray plastique à recycler
+fr:Flacon applicateur, flacon pompe, flacon pistolet/spray plastique à recycler
+fr:Flacon applicateur, flacon pompe, flacon pistolet/spray verre à recycler
```
Comment (teolemon) [inline data/ocr/packaging.txt:240]:
```suggestion
fr:Flacon applicateur à recycler
fr:flacon pompe à recycler
fr:flacon pistolet à recycler
fr:flacon spray à recycler
fr:spray à recycler
fr:Flacon applicateur métal à recycler
fr:flacon pompe métal à recycler
fr:flacon pistolet métal à recycler
fr:flacon spray métal à recycler
fr:spray métal à recycler
fr:Flacon applicateur plastique à recycler
fr:flacon pompe plastique à recycler
fr:flacon pistolet plastique à recycler
fr:flacon spray plastique à recycler
fr:spray plastique à recycler
fr:Flacon applicateur verre à recycler
fr:flacon pompe verre à recycler
fr:flacon pistolet verre à recycler
fr:flacon spray verre à recycler
fr:spray verre à recycler
```

File: data/ocr/packaging.txt [hunk 16]
```diff
@@ -29,40 +73,275 @@ fr:Boîte en métal||boîte metal
...
+fr:Feuille d’emballage (papier kraft, papier sulfurisé, papier cadeau...), feuille plaquette (carton ondulé, fibre) métal à recycler
+fr:Feuille d’emballage (papier kraft, papier sulfurisé, papier cadeau...), feuille plaquette (carton ondulé, fibre) papier à recycler
 fr:Feuille en carton||feuille carton
-fr:Feuille en métal||feuille métal
 fr:Feuille en métal||feuille metal
+fr:Feuille en métal||feuille métal
+fr:Ficelle, ruban, ruban adhésif, bolduc, bande autres à jeter
+fr:Ficelle, ruban, ruban adhésif, bolduc, bande carton à recycler
+fr:Ficelle, ruban, ruban adhésif, bolduc, bande métal à recycler
+fr:Ficelle, ruban, ruban adhésif, bolduc, bande papier à recycler
```
Comment (teolemon) [inline data/ocr/packaging.txt:227]:
```suggestion
fr:Ficelle à jeter
fr:ruban à jeter
fr:ruban adhésif à jeter
fr:bolduc à jeter
fr:bande à jeter
fr:Ficelle carton à recycler
fr:ruban carton à recycler
fr:ruban adhésif carton à recycler
fr:bolduc carton à recycler
fr:bande carton à recycler
fr:Ficelle métal à recycler
fr:ruban métal à recycler
fr:ruban adhésif métal à recycler
fr:bolduc métal à recycler
fr:bande métal à recycler
fr:Ficelle papier à recycler
fr:ruban papier à recycler
fr:ruban adhésif papier à recycler
fr:bolduc papier à recycler
fr:bande papier à recycler
```

File: data/ocr/packaging.txt [hunk 17]
```diff
@@ -29,40 +73,323 @@ fr:Boîte en métal||boîte metal
...
+fr:Seau plastique à jeter
+fr:Seringue pré-remplie plastique à jeter
+fr:Stick autres à jeter
+fr:Stick métal à recycler
+fr:Terrine céramique à jeter
+fr:Terrine grès à jeter
+fr:Tonneau, tonnelet, fût autres à jeter
+fr:Tonneau, tonnelet, fût carton à recycler
+fr:Tonneau, tonnelet, fût métal à recycler
+fr:Tonneau, tonnelet, fût plastique à recycler
```
Comment (teolemon) [inline data/ocr/packaging.txt:374]:
```suggestion
fr:Tonneau à jeter
fr:tonnelet à jeter
fr:fût à jeter
fr:Tonneau carton à recycler
fr:tonnelet carton à recycler
fr:fût carton à recycler
fr:Tonneau métal à recycler
fr:tonnelet métal à recycler
fr:fût métal à recycler
fr:Tonneau plastique à recycler
fr:tonnelet plastique à recycler
fr:fût plastique à recycler
```

File: data/ocr/packaging.txt [hunk 18]
```diff
@@ -29,40 +73,323 @@ fr:Boîte en métal||boîte metal
...
+fr:Cintre autres à jeter
+fr:Cintre carton à recycler
+fr:Cintre métal à recycler
 fr:Clip en plastique;fr:Sachet en plastique||clip et sachet plastique
 fr:Clip en plastique||clip plastique
+fr:Coffre, coffret autre à jeter
+fr:Coffre, coffret carton à recycler
+fr:Coffre, coffret métal à recycler
+fr:Coiffe, collerette métal à recycler
+fr:Coiffe, collerette plastique à jeter
```
Comment (teolemon) [inline data/ocr/packaging.txt:194]:
```suggestion
fr:Coffre à jeter
fr:coffret à jeter
fr:Coffre carton à recycler
fr:coffret carton à recycler
fr:Coffre métal à recycler
fr:coffret métal à recycler
fr:Coiffe métal à recycler
fr:collerette métal à recycler
fr:Coiffe plastique à jeter
fr:collerette plastique à jeter
```

File: data/ocr/packaging.txt [hunk 19]
```diff
@@ -29,40 +73,336 @@ fr:Boîte en métal||boîte metal
...
+fr:flacon spray plastique à recycler
+fr:spray plastique à recycler
+fr:Flacon applicateur verre à recycler
+fr:flacon pompe verre à recycler
+fr:flacon pistolet verre à recycler
+fr:flacon spray verre à recycler
+fr:spray verre à recycler
+fr:Flacon bille / roll on (et son bouchon) plastique à recycler
+fr:Flacon bille / roll on (et son bouchon) verre à recycler
+fr:Flacon bille / roll on (et son bouchon) verre à recycler
```
Comment (teolemon) [inline data/ocr/packaging.txt:280]:
```suggestion
fr:Flacon bille et son bouchon plastique à recycler
fr:Flacon bille et son bouchon verre à recycler
fr:Flacon bille et son bouchon verre à recycler
fr:Flacon roll on et son bouchon plastique à recycler
fr:Flacon roll on et son bouchon verre à recycler
fr:Flacon roll on et son bouchon verre à recycler
fr:Flacon bille (et son bouchon) plastique à recycler
fr:Flacon bille (et son bouchon) verre à recycler
fr:Flacon bille (et son bouchon) verre à recycler
fr:Flacon roll on (et son bouchon) plastique à recycler
fr:Flacon roll on (et son bouchon) verre à recycler
fr:Flacon roll on (et son bouchon) verre à recycler
```

File: data/ocr/packaging.txt [hunk 20]
```diff
@@ -29,40 +73,336 @@ fr:Boîte en métal||boîte metal
...
+fr:Gourde (et son bouchon),doypack, pouch plastique à jeter
+fr:Housse autres à jeter
+fr:Housse carton à recycler
+fr:Housse métal à recycler
+fr:Housse papier à recycler
+fr:Lien *Si l’élément principal est à jeter : lien à jeter à jeter
+fr:Lien *Si l’élément principal est à recycler : lien à recycler à recycler
+fr:Malette, valise, valisette carton à recycler
+fr:Malette, valise, valisette métal à recycler
+fr:Malette, valise, valisette plastique à jeter
```
Comment (teolemon) [inline data/ocr/packaging.txt:296]:
```suggestion
fr:lien à jeter
fr:lien à recycler
fr:Malette carton à recycler
fr:valise carton à recycler
fr:valisette carton à recycler
fr:Malette métal à recycler
fr:valise métal à recycler
fr:valisette métal à recycler
fr:Malette plastique à jeter
fr:valise plastique à jeter
fr:valisette plastique à jeter
```

File: data/ocr/packaging.txt [hunk 21]
```diff
@@ -29,40 +73,351 @@ fr:Boîte en métal||boîte metal
...
+fr:Malette carton à recycler
+fr:valise carton à recycler
+fr:valisette carton à recycler
+fr:Malette métal à recycler
+fr:valise métal à recycler
+fr:valisette métal à recycler
+fr:Malette plastique à jeter
+fr:valise plastique à jeter
+fr:valisette plastique à jeter
+fr:Moule autres à jeter
```
Comment (teolemon) [inline data/ocr/packaging.txt:312]:
```suggestion
fr:Moule à jeter
```

File: data/ocr/packaging.txt [hunk 22]
```diff
@@ -3,24 +3,68 @@
...
+es:Botella||embotellado
 es:Envase en cartón||envase cartón
 es:Envase en metal||envase metal
+fr:Aérosol métal à recycler
+fr:Ailettes papier à recycler
+fr:Ailettes plastique à jeter
+fr:Ampoule verre à recycler
+fr:Applicateur carton à recycler
+fr:Applicateur plastique à jeter
+fr:Bac autres à jeter
```
Comment (teolemon) [inline data/ocr/packaging.txt:20]:
```suggestion
fr:Bac à jeter
```

File: data/ocr/packaging.txt [hunk 23]
```diff
@@ -29,40 +73,351 @@ fr:Boîte en métal||boîte metal
...
+fr:éléments de mise en forme carton à recycler
+fr:éléments de maintien des produits carton à recycler
+fr:Cale plastique à jeter
+fr:coin plastique à jeter
+fr:cornière plastique à jeter
+fr:éléments de bourrage plastique à jeter
+fr:éléments de mise en forme plastique à jeter
+fr:éléments de maintien des produits plastique à jeter
+fr:Canette métal à recycler
+fr:Capsule autres à jeter
```
Comment (teolemon) [inline data/ocr/packaging.txt:161]:
```suggestion
fr:Capsule à jeter
```

File: data/ocr/packaging.txt [hunk 24]
```diff
@@ -29,40 +73,351 @@ fr:Boîte en métal||boîte metal
...
+fr:cornet carton à recycler
+fr:Cavalier métal à recycler
+fr:étui métal à recycler
+fr:fourreau métal à recycler
+fr:cornet métal à recycler
+fr:Cavalier papier à recycler
+fr:étui papier à recycler
+fr:fourreau papier à recycler
+fr:cornet papier à recycler
+fr:Cintre autres à jeter
```
Comment (teolemon) [inline data/ocr/packaging.txt:185]:
```suggestion
fr:Cintre à jeter
```

File: data/ocr/packaging.txt [hunk 25]
```diff
@@ -29,40 +73,351 @@ fr:Boîte en métal||boîte metal
...
+fr:coffret carton à recycler
+fr:Coffre métal à recycler
+fr:coffret métal à recycler
+fr:Coiffe métal à recycler
+fr:collerette métal à recycler
+fr:Coiffe plastique à jeter
+fr:collerette plastique à jeter
+fr:Collier plastique à jeter
+fr:Coque en plastique||coque plastique
+fr:Couvercle autres à jeter
```
Comment (teolemon) [inline data/ocr/packaging.txt:202]:
```suggestion
fr:Couvercle à jeter
```

File: data/ocr/packaging.txt [hunk 26]
```diff
@@ -29,40 +73,351 @@ fr:Boîte en métal||boîte metal
...
+fr:Couvercle verre à recycler
+fr:Dose individuelle autres à jeter
+fr:Dose individuelle carton à recycler
+fr:Dose individuelle papier à recycler
+fr:Enveloppage autres à jeter
+fr:Enveloppage métal à recycler
+fr:Enveloppage papier à recycler
+fr:Enveloppe carton à recycler
+fr:Enveloppe papier à recycler
+fr:Étiquette autres à jeter
```
Comment (teolemon) [inline data/ocr/packaging.txt:217]:
```suggestion
fr:Dose individuelle à jeter
fr:Dose individuelle carton à recycler
fr:Dose individuelle papier à recycler
fr:Enveloppage à jeter
fr:Enveloppage métal à recycler
fr:Enveloppage papier à recycler
fr:Enveloppe carton à recycler
fr:Enveloppe papier à recycler
fr:Étiquette à jeter
```

File: data/ocr/packaging.txt [hunk 27]
```diff
@@ -29,40 +73,351 @@ fr:Boîte en métal||boîte metal
...
+fr:ruban métal à recycler
+fr:ruban adhésif métal à recycler
+fr:bolduc métal à recycler
+fr:bande métal à recycler
+fr:Ficelle papier à recycler
+fr:ruban papier à recycler
+fr:ruban adhésif papier à recycler
+fr:bolduc papier à recycler
+fr:bande papier à recycler
+fr:Filet autres à jeter
```
Comment (teolemon) [inline data/ocr/packaging.txt:249]:
```suggestion
fr:Filet à jeter
```

File: data/ocr/packaging.txt [hunk 28]
```diff
@@ -29,40 +73,351 @@ fr:Boîte en métal||boîte metal
...
+fr:Flacon roll on (et son bouchon) verre à recycler
 fr:Flacon en plastique||flacon plastique
-HDPE||HDPE
+fr:Gobelet autres à jeter
+fr:Gobelet carton à recycler
+fr:Gobelet métal à recycler
+fr:Gobelet papier à recycler
+fr:Gobelet verre à recycler
+fr:Gourde (et son bouchon),doypack, pouch plastique à jeter
+fr:Housse autres à jeter
```
Comment (teolemon) [inline data/ocr/packaging.txt:297]:
```suggestion
fr:Gobelet à jeter
fr:Gobelet carton à recycler
fr:Gobelet métal à recycler
fr:Gobelet papier à recycler
fr:Gobelet verre à recycler
fr:Gourde (et son bouchon) à jeter
fr:Gourde et son bouchon à jeter
fr:Gourde à jeter
fr:doypack à jeter
fr:pouch plastique à jeter
fr:Housse à jeter
```

File: data/ocr/packaging.txt [hunk 29]
```diff
@@ -29,40 +73,351 @@ fr:Boîte en métal||boîte metal
...
+fr:pochette métal à recycler
+fr:Sac papier à recycler
+fr:sachet papier à recycler
+fr:poche papier à recycler
+fr:pochette papier à recycler
+fr:Sachet en plastique||sachet plastique
+fr:Seau métal à recycler
+fr:Seau plastique à jeter
+fr:Seringue pré-remplie plastique à jeter
+fr:Stick autres à jeter
```
Comment (teolemon) [inline data/ocr/packaging.txt:387]:
```suggestion
fr:Stick à jeter
```

File: data/ocr/packaging.txt [hunk 30]
```diff
@@ -29,40 +73,351 @@ fr:Boîte en métal||boîte metal
...
 fr:Pot en carton||pot carton
 fr:Pot en plastique;fr:Opercule en plastique||pot et opercule plastique
 fr:Pot en plastique||pot plastique
 fr:Pot en verre||pot verre
+fr:Pot métal à recycler
+fr:Pot plastique à jeter
+fr:Pot verre à recycler
+fr:Présentoir, socle, casier, chevalet autres à jeter
+fr:Présentoir, socle, casier, chevalet carton à recycler
+fr:Présentoir, socle, casier, chevalet métal à recycler
```
Comment (teolemon) [inline data/ocr/packaging.txt:369]:
```suggestion
fr:Présentoir à jeter
fr:socle à jeter
fr:casier à jeter
fr:chevalet à jeter
fr:Présentoir carton à recycler
fr:socle carton à recycler
fr:casier carton à recycler
fr:chevalet carton à recycler
fr:Présentoir métal à recycler
fr:socle métal à recycler
fr:casier métal à recycler
fr:chevalet métal à recycler
```

File: data/ocr/packaging.txt [hunk 31]
```diff
@@ -29,40 +73,364 @@ fr:Boîte en métal||boîte metal
...
+fr:Bonbonne métal à recycler
+fr:Bonbonne plastique à recycler
+fr:Bonbonne verre à recycler
+fr:Bouchon en plastique||bouchon plastique
+fr:Bouchon tous matériaux
+fr:Bouteille (et son bouchon)
+fr:Bouteille (et son bouchon) métal à recycler
+fr:Bouteille (et son bouchon) plastique à recycler
+fr:Bouteille (et son bouchon) verre
+fr:Bouteille (et son bouchon) verre
```
Comment (teolemon) [inline data/ocr/packaging.txt:107]:
```suggestion
fr:Bouteille (et son bouchon)
fr:Bouteille (et son bouchon) métal à recycler
fr:Bouteille (et son bouchon) plastique à recycler
fr:Bouteille (et son bouchon) verre
fr:Bouteille (et son bouchon) verre
fr:Bouteille et son bouchon
fr:Bouteille métal et son bouchon à recycler
fr:Bouteille plastique et son bouchon à recycler
fr:Bouteille en verre et son bouchon
```

**Comment by teolemon** [comment]
=========================== short test summary info ============================
FAILED tests/unit/prediction/ocr/test_packaging.py::test_packaging_format - A...
FAILED tests/unit/prediction/ocr/test_packaging.py::test_find_packaging[Tetrapack-value_tags0]
FAILED tests/unit/prediction/ocr/test_packaging.py::test_find_packaging[tetrapack-value_tags1]
FAILED tests/unit/prediction/ocr/test_packaging.py::test_find_packaging[Packaging: bo\xeete carton,...-value_tags2]
FAILED tests/unit/prediction/ocr/test_packaging.py::test_find_packaging[Ingr\xe9dients: 100% tomate-value_tags3]
FAILED tests/unit/prediction/ocr/test_packaging.py::test_find_packaging[emballage: bouteille verre et son bouchon...-value_tags5]
============ 6 failed, 237 passed, 15 warnings in 118.98s (0:01:58) ============

File: data/ocr/packaging.txt [hunk 32]
```diff
@@ -3,24 +3,68 @@
 21-PAP||21-PAP
 21-PAP||21-PAP
```
Comment (teolemon) [inline data/ocr/packaging.txt:4]:
```suggestion
```

Comment (teolemon) [reply data/ocr/packaging.txt:4]: am i missing smg ?

Comment (mithridatea) [reply data/ocr/packaging.txt]: no there used to be duplicates

**Review by mithridatea — APPROVED** [review]

**Pull request merged**